mw/xml_dump/iteration/page.py

```diff
@@ -25,10 +25,9 @@ class Page(serializable.Type):
         'namespace',
         'redirect',
         'restrictions',
-        'revisions'
     )
 
-    def __init__(self, id, title, namespace, redirect, restrictions, revisions):
+    def __init__(self, id, title, namespace, redirect, restrictions, revisions=[]):
```

Comment:
This is dangerous.  This can potentially cause some bad behavior.  It turns out that all page.revisions will reference this one list.  Here's an example of something that could go wrong:

>>> def foo(bar=[]):
...     return bar
... 
>>> foo()
[]
>>> foo().append(5)
>>> foo()
[5]